tests/test_torch_nn.py

```diff
@@ -184,6 +184,28 @@ def test_torch_nn_Dropout():
   assert jnp.mean(res_jax == res_torch.numpy(force=True)) > 0.5
 
 
+def test_torch_nn_PReLU():
```

Comment:
nit: let's keep these tests alphabetized, so this would go after `test_torch_nn_MaxPool2d`